python/cog/director/director.py

```diff
@@ -162,7 +167,16 @@ def _loop(self):
             structlog.contextvars.bind_contextvars(message_id=message_id)
 
             try:
-                self._handle_message(message_id, message_json)
+                span_attributes = {
+                    "idling_ms": (
```

Comment:
I added this, but I don't think it's going to help. Even emitting a span for the duration where we're idle also doesn't do much. It's just an unfortunate limitation of Honeycomb to my knowledge, and will also not give us the most accurate utilization we could get. That said, it is a sampling every 15s, which is not awful.

Reply:
Do we even still need this?